lib/stm32wba/Common/WPAN/Modules/scm.h

```diff
@@ -16,7 +15,6 @@
   *
   ******************************************************************************
   */
-/* USER CODE END Header */
```

Comment:
This is not strictly need, I would prefer to minimize changes in this PR.
This is valid for all changes in this file.